src/WatchDogs.FakeSource/IDataGeneratorRepeaterBackroundTask.cs

```diff
@@ -0,0 +1,8 @@
+﻿
+namespace WatchDogs.FakeSource;
+
+public interface IDataGeneratorRepeaterBackroundTask
+{
+    void Start();
+    Task StopAsync();
```

Comment:
And if we pass cancellation token as above, we can remove the `StopAsync` and do what it does when the cancellation token is cancelled.

Reply:
Done.